applications/poisson/sine/input.json

```diff
@@ -7,7 +7,7 @@
     "Resolution": {
         "RunType": "RefineHAndP",
         "DegreeMin": "1",
-        "DegreeMax": "15",
+        "DegreeMax": "2",
```

Comment:
The default setup should be Hypercube (with Multigrid), so that we do not have to change this parameter.

Reply:
This is probably the reason why the tests failed. So we should undo these changes so that we can merge the rest of this PR.